docs/manage/security/saml/set-up-saml.md

```diff
@@ -62,32 +61,28 @@ Before provisioning SAML, make sure you have the following:
 Follow these steps to configure IdP-initiated login. After this procedure, you can enable optional SAML functionality, including SP-initiated login and on-demand provisioning, as described in [Optional Configurations](set-up-saml.md).
 
 1. Go to **Administration** > **Security** > **SAML**.
-1. Select an existing configuration, or click the plus (**+**) icon to create a new configuration.
-
-    ![saml-config-list.png](/img/security/saml-config-list.png) 
-
-1. The **Add Configuration** page appears.
-1. **Configuration Name**: Enter a name to identify the SSO policy (or another name used internally to describe the policy).
-1. **Debug Mode**: Select this option if you'd like to view additional details if an error occurs when a user attempts to authenticate. For more information, see [View SAML Debug Information](view-saml-debug-information.md).
-1. **Issuer**: Enter the unique URL assigned to your organization by the SAML IdP. 
-
-    ADFS example: `http://adfs.myserver.tld/adfs/services/trust`
-
-1. **X.509 Certificate**: Copy and paste your organization's X.509 certificate, which is used to verify signatures in SAML assertions. For ADFS, the certificate required is the Token-signing ADFS X.509 certificate.
-1. **Attribute Mapping**: Depending on your IdP, select: 
-
+1. Select an existing configuration, or click the plus (**+**) icon to create a new configuration. <br/>![saml-config-list.png](/img/security/saml-config-list.png) 
+1. The **Add Configuration** page appears.<br/><img src={useBaseUrl('img/security/saml-add-configuration.png')} alt="Add configuration" style={{border: '1px solid black'}} width="600" />
+1. **Configuration Name**. Enter a name to identify the SSO policy (or another name used internally to describe the policy).
+1. **Debug Mode**. Select this option if you'd like to view additional details if an error occurs when a user attempts to authenticate. For more information, see [View SAML Debug Information](view-saml-debug-information.md).
+1. **Issuer**. Enter the unique URL assigned to your organization by the SAML IdP. <br/>ADFS example: `http://adfs.myserver.tld/adfs/services/trust`
+1. **X.509 Certificate**:.Copy and paste your organization's X.509 certificate, which is used to verify signatures in SAML assertions. For ADFS, the certificate required is the Token-signing ADFS X.509 certificate.
+1. **Attribute Mapping**. Depending on your IdP, select: 
```

Comment:
@jpipkin1 just wanted to let you know I changed the colons to periods here. It wasn't part of your PR, but thought I would take care of them. Everything else looks good.

Reply:
Thanks for doing that cleanup, @kimsauce.